Windows LLVM clang LINK warning

```
LINK : warning LNK4217: symbol '__std_terminate' defined in 'libvcruntime.lib(ehhelpers.obj)' is imported by 'output_env_var_tests-b23a65.o' in function '"int `void __cdecl std::_Deallocate<16,0>(void *,unsigned __int64)'::`1'::dtor$4" (?dtor$4@?0???$_Deallocate@$0BA@$0A@@std@@YAXPEAX_K@Z@4HA)'
LINK : warning LNK4217: symbol '_CxxThrowException' defined in 'libvcruntime.lib(throw.obj)' is imported by 'output_env_var_tests-b23a65.o' in function '"public: void __cdecl std::ios_base::clear(int,bool)" (?clear@ios_base@std@@QEAAXH_N@Z)'

```

--- a/utils/clangversion.mk
+++ b/utils/clangversion.mk
@@ -27,6 +27,7 @@ ifdef IUTEST_CLANG_CXX
 
 CXX_NAME=clang++
 CLANGVERSION:=$(shell $(IUTEST_CLANG_CXX) --version | grep version | sed "s/.*version[ ]*\([0-9]*\.[0-9]*\).*/\1/")
+CLANG_TATGET:=$(shell $(IUTEST_CLANG_CXX) --version | grep Target | sed "s/.*Target:[ ]*\(.*\)/\1/")
 
 dot:=.
 empty:=
@@ -123,6 +124,10 @@ IUTEST_CXX_WARN_FLAGS+=-Wno-missing-field-initializers
 # 8.0 later
 ifeq (1,$(shell expr \( $(CLANGMAJOR) \>= 8 \)))
 IUTEST_CXX_WARN_FLAGS+=-Wextra-semi
+endif
+
+ifeq ($(CLANG_TATGET), x86_64-pc-windows-msvc)
+CXXFLAGS+= -Xclang -flto-visibility-public-std
 endif
 
 endif

